reference/cli/sailsdebug.md

```diff
@@ -23,6 +23,26 @@ debugger listening on port 5858
 
 > To use the standard (command-line) node debugger with sails, you can always just run `node debug app.js`.
 
+### Use node-inspector
```

Comment:
Hey @rdroro, thanks for adding this! I've highlighted a few spelling nitpicks line-by-line. Once these are fixed, I'll merge your changes and literally transform into the joy cat 😹